### ISSUE TITLE
[announcement] csv2exif repository is online

### ISSUE THREAD

**Issue by mrtngrsbch** [hvdwolf/jExifToolGUI#248]
Hi,

I am pleased to announce that I have published the script 'csv2exif' [https://github.com/MuseosAbiertos/csv2exif].
'csv2exif' records metadata on images (ex. jpg) from a normalized CSV using VRAE, ISADG & Dublin Core standards.

feedback welcome

**Comment by hvdwolf** [comment]
Very nice. 
When running the csv-demo on the images-demo folder, I get quite some errors like `[!] Row "126": File not found in this path: "AR-MA-Bruzzone-Bianchic-Zaffaroni-Nailture-3-1284-00.jpg"`.
The image is indeed not found, but the script nicely continues.
For the rest it seems to work fine, but I am not an VRAE, ISADG & Dublin Core expert.

**Comment by mrtngrsbch** [comment]
You will indeed see a lot of errors, which is normal because not all images are available for you to see the errors.

The log is very detailed and also records a 'success_log' and 'error_log'.
Thank you very much for testing it. As you can see, you can adapt it to many standards; in the 'maps.json' file you can configure the data mapping.

This tool is very useful when you deal with a lot of files. In the case of demo there are 2000, but locally I have up to 14.000 !

**Comment by hvdwolf** [comment]
Are you planning a UI "in front" of it?

**Comment by mrtngrsbch** [comment]
Sincerely, the tool has such a simple interface that I hadn't thought about it, besides my budget was devastated with the UST crash. 
I am also making other tools, such as [https://github.com/MuseosAbiertos/docx2csv].

Can you think of anything ?

**Comment by hvdwolf** [comment]
Your budget? 
Did you write it or some python programmer?

PySimpleGUI is an easy to make UI but misses some high-end features, which is not necessary currently for your app. I discovered it 2-3 months ago. It is what I used for my latest [PyImageFuser](https://github.com/hvdwolf/PyImageFuser). 
After I finished my 1 year work project, I was not motivated yet to continue with jExifToolGUI, but I wanted to do "something different", so I decided to go back in time and rewrite my old [ImageFuser](http://imagefuser.sourceforge.net/) in python with PySimpleGui. 
If PySimpleGUI had existed by the time I gave up on PyExifToolGUI as PySide was almost dead, then jExifToolGUI would not have existed but I would have rewritten PyExifToolGUI in python/pysimplegui. Next to that: also PySide has come to life again and is also supported by PySimpleGUI.

**Comment by hvdwolf** [comment]
With PySimpleGUI you could something like this. I created this in 10 minutes, but I did not create the actions yet.
![Screenshot 2022-06-12 10 43 30](https://user-images.githubusercontent.com/4520361/173225035-c7b8f368-6c08-4de2-8ebe-76477f720a8c.png)

**Comment by hvdwolf** [comment]
This is now the most simple version possible.
No error checking.
Optional arguments to defaults
To be started with `python3 gcsv2exif.py`
Can be made to an executable version for MacOS, Linux and Windows (But I did absolutely nothing in that direction)

https://user-images.githubusercontent.com/4520361/173234848-b8f03c86-f8f5-4a00-bffe-d6560709ffc3.mp4

**Comment by mrtngrsbch** [comment]
Yes, a python programmer, for three reasons; I'm not good at programming yet, I'm still learning python, I have several simultaneous projects, like you.

So my main task is, at the moment, to create the program and requirements, to document well and to be a good owner. All this is to create a stack of tools for collection management and my proposal for a "lightweight" Digital Asset Management to be visualized in Notion. ex.: [https://martingggg.notion.site/cba074312276413cae988fda23e2580c?v=7d4341458cfb4aa39f0e38b57e8948e5]

OMG, PyImageFuser is very interesting ! I will try it later with my images. I have a Fujifilm x100s, but I use a lot my OnePlus which takes RAW photos and has HDR.
https://martin.gersbach.net/

True, when you changed Python to Java I wondered why you did it. That was a lot of effort. I've been looking into Python GUIs and PySimpleGUI is my favorite. Indeed I have to dedicate more time to my studies.

Heh... you have already created in a few minutes a GUI of csv2exif ! That sounds incredible. Do you plan to make your own repository or do you want to make a merge with csv2exif?

I am always very grateful for your help with these topics.

**Comment by hvdwolf** [comment]
> Heh... you have already created in a few minutes a GUI of csv2exif ! That sounds incredible. Do you plan to make your own repository or do you want to make a merge with csv2exif?
> 
> I am always very grateful for your help with these topics.

I will do some further additions and make it a pull request to your repository. The gui was created in 10-15 minutes. I kept the gui code separate from the programming code (apart from the progress window updates of course). Then making it work with the original scripts took about 1½ hours.  Your python programmer is a really an Object Oriented programmer. Being old it takes some time for me to adapt ;)

**Comment by mrtngrsbch** [comment]
cool...
"If you think you are smarter than previous generations, remember that 50 years ago the car owner's manual indicated how to regulate the valves, today it warns you not to ingest the battery fluid."

**Comment by hvdwolf** [comment]
I just created a pull request.
I also updated the README and corrected it for official "2 space" line endings as markdown wants it.

**Comment by mrtngrsbch** [comment]
1st issue ;-)
https://github.com/MuseosAbiertos/csv2exif/issues/2

**Comment by mrtngrsbch** [comment]
Just happy ;-)

![ScreenShot 2022-06-12 a las 13 22 39](https://user-images.githubusercontent.com/693328/173242997-7b7680a6-442a-40b3-9109-433743079cc8.jpg)

**Comment by hvdwolf** [comment]
> 1st issue ;-) [MuseosAbiertos/csv2exif#2](https://github.com/MuseosAbiertos/csv2exif/issues/2)
> 
> * pip3 install PySimpleGUI

That's why I updated the README with the section "requirements"
